### ISSUE TITLE
Support for Fenstermelder free@home Wireless

### ISSUE THREAD

**Issue by Alox-K** [henry-spanka/homebridge-freeathome#33]
Hi Henry,

i added a "6222/1 AP-64-WL Fenstermelder free@home Wireless".
In the free@home-System, it workes, but it seems I have no access (status display) of this in Homekit.
Are these deviced supported? 
Or am I doning something wrong (since this is the first time, I added devices since I'm using hoobs...

Thank you!

**Comment by henry-spanka** [comment]
They should work according to other users. Please send me the backup from the SysAp for me to look at. Thanks

**Comment by Alox-K** [comment]
Hey!

I send you the backup from my sysap in the issue reagarding the trigger actuators.
Maybe this gives you an idea, why my Fenstermelder doesn’t work.

Or do I Need to add them on some way to hoobs, after I configured them in the sysAP? If so, how does it work?



Von: Henry Spanka
Gesendet: Sonntag, 11. Oktober 2020 15:09
An: henry-spanka/homebridge-freeathome
Cc: Alox-K; Author
Betreff: Re: [henry-spanka/homebridge-freeathome] Support for Fenstermelderfree@home Wireless (#33)

They should work according to other users. Please send me the backup from the SysAp for me to look at. Thanks

**Comment by henry-spanka** [comment]
Where did you send it? I can not find it in my emails.

**Comment by Alox-K** [comment]
I sent it again...

**Comment by Alox-K** [comment]
Die you recieve my mail?

**Comment by henry-spanka** [comment]
Is this still relevant?

**Comment by floriangrotz** [comment]
Yes it is - if I can help feel free to contact me - I can delivery anything you need. 
I´m wondering as the "Universalmelder" from BJ is exposed.

**Comment by floriangrotz** [comment]
Hello @henry-spanka - still open - so Universalmelder wireless is working, but Fenstermelder wireless do not show up. 
Should I send you something?
